src/BookNavigator/volumes/volumes.js

```diff
@@ -0,0 +1,178 @@
+import { css, html, LitElement } from 'lit-element';
+import { nothing } from 'lit-html';
+import { repeat } from 'lit-html/directives/repeat';
+
+export class Volumes extends LitElement {
+  static get properties() {
+    return {
+      subPrefix: { type: String },
+      hostUrl: { type: String },
+      viewableFiles: { type: Array },
+    };
+  }
+
+  constructor() {
+    super();
+    this.subPrefix = '';
+    this.hostUrl = '';
+    this.viewableFiles = [];
+  }
+
+  firstUpdated() {
+    const activeFile = this.shadowRoot.querySelector('.content.active');
+    // allow for css animations to run before scrolling to active file
+    setTimeout(() => {
+      // scroll active file into view if needed
+      // note: `scrollIntoViewIfNeeded` handles auto-scroll gracefully for Chrome, Safari
+      // Firefox does not have this capability yet as it does not support `scrollIntoViewIfNeeded`
+      if (activeFile?.scrollIntoViewIfNeeded) {
+        activeFile?.scrollIntoViewIfNeeded(true);
+        return;
+      }
+
+      // Todo: support `scrollIntoView` or `parentContainer.crollTop = x` for FF & "IE 11"
+      // currently, the hard `position: absolutes` misaligns subpanel when `scrollIntoView` is applied :(
+    }, 350);
+  }
+
+  volumeItemWithImageTitle(item) {
```

Comment:
Is this being used anywhere?

Reply:
not yet. just UI prep for when we do get those images as we created UI per spec